_posts/2020-06-23-sourcify-faq.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: post
+published: true
+title:  "All you need to know about Sourcify"
+date:   2020-06-23
+author: Franziska Heintel
+category: Explainers
+---
+
+Welcome to Sourcify's first short FAQ! If your questions around source verification haven't been answered after reading 
+this post, please feel free to drop by the [Sourcify Gitter channel](https://gitter.im/ethereum/source-verify) and ask us 
+any question there. Also stay tuned on more turorials and developer focused content to follow here and on the [Remix blog](https://medium.com/remix-ide)!
+
+## Source Verification
+
+### What is source verification?
+
+In a software development context, the term source verification describes the process of verifying the legitimacy of code 
```

Comment:
"legitimacy" sounds a bit weird to me

Reply:
What would you suggest?

Reply:
truthfulness? validity?

Reply:
I would actually say that this is not a term that is used in software development in general, but only in our blockchain context. So maybe we could remove the generic first sentence and start with the "special" case?

Reply:
That works for me. Deleted the first sentence.